tool/site/client-react/src/pages/sendPostSignPage.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react"
+import {useContext} from "react"
+import {Button, CrossIcon, Label, SummaryList, Table, TickIcon} from "nhsuk-react-components"
+import axios from "axios"
+import * as fhir from "fhir/r4"
+import MessageExpanders from "../components/messageExpanders"
+import ButtonList from "../components/buttonList"
+import LongRunningTask from "../components/longRunningTask"
+import {AppContext} from "../index"
+import PrescriptionActions from "../components/prescriptionActions"
+
+interface SendPostSignPageProps {
+  token: string
+}
+
+const SendPostSignPage: React.FC<SendPostSignPageProps> = ({
+  token
+}) => {
+  const {baseUrl} = useContext(AppContext)
+  const sendPrescriptionTask = () => sendPrescription(baseUrl, token)
+  return (
+    <LongRunningTask<SendResult | SendBulkResult> task={sendPrescriptionTask} loadingMessage="Sending prescription(s).">
+      {sendResult => {
+        if (isBulkResult(sendResult)) {
+          return <>
+            <Label isPageHeading>Send Results</Label>
+            <ButtonList>
+              <Button
+                onClick={() => navigator.clipboard.writeText(sendResult.results.map(r => r.prescription_id).join("\n"))}>Copy
```

Comment:
whats up with the newlines here?

Reply:
You're usually going to paste this list into Slack or an email

Reply:
Oh you meant the newlines in the code :facepalm: